static/src/js/components/CustomSelectWidget/CustomSelectWidget.jsx

```diff
@@ -96,6 +97,8 @@ const CustomSelectWidget = ({
     value: enumValue
   }))
 
+  const { enumOptions: oneOfEnums } = options || {}
```

Comment:
I would maybe move this line right about line 120 just to keep it closer to the logic using it.
```suggestion
  const { enumOptions: oneOfEnums } = options || {}
```
```suggestion
  const { enumOptions: oneOfEnums } = options || {}
```